Fatal error: exception Failure("Function my_func had a non-void return but had branches that never returned.")
